MapboxNavigation/NavigationMapView.swift

```diff
@@ -1025,39 +1027,122 @@ extension Dictionary where Key == Int, Value: MGLStyleValue<NSNumber> {
 }
 
 //MARK: NavigationMapViewDelegate
-
+/**
+ The `NavigationMapViewDelegate` provides methods for configuring the NavigationMapView, as well as responding to events triggered by the NavigationMapView.
+ */
 @objc(MBNavigationMapViewDelegate)
 public protocol NavigationMapViewDelegate: class  {
+    /**
+     Asks the reciever to return an MGLStyleLayer for routes, given an identifier and source.
+     - note: This method is invoked at initalization time and anytime routes are added.
+     - parameter mapView: the NavigationMapView.
+     - parameter identifier: the style identifier.
+     - parameter source: The Layer source provided.
+     - returns: an MGLStyleLayer that the map applies to all routes.
```

Comment:
Nit: Let’s be consistent and capitalize the first word to all these items, even if they are sentence fragments.